.github/workflows/pr-actions.yaml

```diff
@@ -4,15 +4,37 @@ on:
     types: [created, edited]
 
 jobs:  
+  parse:
+    runs-on: ubuntu-latest
+    name: Parse PR Comment
+    if: github.event.issue.pull_request
+    outputs:
+      command_name: ${{ steps.extract.outputs.command_name }}
+    steps:
+      - name: Extract command from comment
+        id: extract
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const raw = context.payload.comment?.body ?? '';
+            const lines = String(raw)
+              .split(/\r?\n/)
+              .map(l => l.trim())
+              .filter(l => l.length > 0);
+            const allowed = new Set(['/publish', '/update-versions', '/update-commit']);
+            const firstMatching = lines.find(l => allowed.has(l)) || '';
```

Comment:
Shouldn't we fail with a clear message if several actions are added as comments, instead of ignoring the additional actions ?